app/src/app/api/v0/city/[city]/user/route.ts

```diff
@@ -0,0 +1,24 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { createUserRequest } from "@/util/validation";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+import { randomUUID } from "node:crypto";
+
+export const POST = apiHandler(async (_req: NextRequest, { params }) => {
+  const body = createUserRequest.parse(await _req.json());
+  const user = await db.models.User.create({
+    userId: randomUUID(),
+    ...body,
+  });
```

Comment:
If we create a user this way, it won't have any login information attached. So it's maybe required to allow for the user to sign up afterwards still (which will probably fail if a record with their email exists already). So if we use this we need to change the signup endpoint as well (and perhaps set an `isInvited` boolean field in the user record here to show that it's not a fully registered user yet?).